build: add support for node 16

iamsobanjaved opened the pull request:
Adding support for node 16

### Screenshots:

Courses page:

<img width="1918" alt="Screenshot 2022-04-20 at 11 59 08 PM" src="https://user-images.githubusercontent.com/58461728/164303876-8499c4a3-c42f-4282-8e6c-9c03377ad261.png">

No error reported on console:

<img width="1919" alt="Screenshot 2022-04-20 at 11 59 30 PM" src="https://user-images.githubusercontent.com/58461728/164303859-431b7ae8-c59f-4ce8-bdb1-d73f3d8885a1.png">

awais786 commented on .github/workflows/ci.yml:
```diff
@@ -43,14 +44,13 @@ jobs:
           TARGETS: "quality"
       - name: test js
-        run: ./.github/scripts/testing.sh
+        run: ./.github/scripts/testing-js.sh
```
node16 is brining new translations. One solution might be merge this PR and then run transifex job and then enable this check again.

schenedx commented:
I reviewed all and my biggest concern is on the upgrade of `babel` libraries upgrading to `@babel` versions.
Have you tried to run insights dashboard frontend using this branch? Are the UI impacted by looking at it?

iamsobanjaved commented:
@schenedx I have added screenshots, you can verify if UI is fine by these pics. As we have upgraded babel and changed to `@babel`, this was causing issues that delayed this. I forgot to update the name of one of the babel plugins in the webpack config. But as now build command is working after that fix, I think we are safe now.

schenedx reviewed (approved):
@iamsobanjaved Alright... Let's try this. 👍 
Feel free to merge. I can test on stage as well.

awais786 commented:
@schenedx stage is looking fine.
https://stage-insights.edx.org/

schenedx commented:
@awais786 Yes. It looks like this is successful. Want to go ahead and deploy the changes to prod?

awais786 commented:
@schenedx does this pr and config PR need to backport into nutmeg ?
